2996_reflection/reflection.md

```diff
@@ -5047,14 +5016,18 @@ Use "host scope" in lieu of "inhabits" in paragraph 4:
 
 :::
 
-Modify paragraph 9 to allow reflections of non-static data members to appear in default function arguments, extend example 8 which follows, and use "host scope" rather than "inhabits" following example 9.
+Modify paragraph 9 to allow reflections of non-static data members to appear in default function arguments, extend example 8 which follows, and use "host scope" rather than "inhabits" following example 9. Break the list of exemptions in paragraph 9 into bullets for better readability.
 
 ::: std
 [9]{.pnum} A default argument is evaluated each time the function is called with no argument for the corresponding parameter.
 
 [...]
 
-A non-static member shall not appear [or be designated]{.addu} in a default argument unless it appears as the `$id-expression$` [or `$splice-expression$`]{.addu} of a class member access expression ([expr.ref]) or unless it is used to form a pointer to member ([expr.unary.op]) [or a reflection ([expr.reflect])]{.addu}.
+A non-static member shall not [appear]{.rm} [be designated]{.addu} in a default argument unless[\ ]{.addu}
```

Comment:
Don't need to markup the space lol

Reply:
Lmao leave me alone; I do it when I'm breaking a sentence into bullets 😂 And it's consistent with where I've done that elsewhere already haha